testnet/run_libp2p.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -f /usr/bin/etcdctl ]; then
+	./etc/nomos/etcd/install_etcd.sh
+fi
+
+
+NET_NODE_KEY=$(./etc/nomos/etcd/register_etcd.sh)
+CONSENSUS_NODE_KEY=$NET_NODE_KEY
+
+sleep 2
```

Comment:
A wild sleep appeared? Why? At least add a comment explaining why this is here.

Reply:
No excuses for wild sleep, will remove it completely.